mongo/integration/json_helpers_test.go

```diff
@@ -26,13 +26,16 @@ import (
 )
 
 var (
-	awsAccessKeyID     = os.Getenv("AWS_ACCESS_KEY_ID")
-	awsSecretAccessKey = os.Getenv("AWS_SECRET_ACCESS_KEY")
-	azureTenantID      = os.Getenv("AZURE_TENANT_ID")
-	azureClientID      = os.Getenv("AZURE_CLIENT_ID")
-	azureClientSecret  = os.Getenv("AZURE_CLIENT_SECRET")
-	gcpEmail           = os.Getenv("GCP_EMAIL")
-	gcpPrivateKey      = os.Getenv("GCP_PRIVATE_KEY")
+	awsAccessKeyID         = os.Getenv("AWS_ACCESS_KEY_ID")
```

Comment:
The `client_side_encryption_spec_test.go` file has a `verifyClientSideEncryptionVarsSet` function to assert that the existing seven aws/azure/gcp variables are non-empty. Should we also add these three new variables to that function? Are there any cases where they're expected to be unset?

Reply:
I think they should always be set, so it's a good idea to add them! Added in a check for each in `verifyClientSideEncryptionVarsSet`.

Reply:
Actually, the three new temp variables are not set on 2.6 and 3.0 server versions during evg tests (this is because the `PYTHON3_BINARY` env var is not set), so I can't check them here. The `verifyClientSideEncryptionVarsSet` should really not be running on anything less than `4.1.10`, but I think I'd need to switch to using `mtest` to use the version conditionally... Happy to do that, or just remove the checks for now.

Reply:
Discussed offline: checked for the existence of the three new variables when they are used in `json_helpers_test`. Documented this in `verifyClientSideEncryptionVarsSet`.